generator/templates/coverage-matrix/styles.css

```
@@ -0,0 +1,361 @@
+    color: #2e7d32;
+    margin: 16px 0 8px;
+}
+
+.appendix-list {
+    margin-left: 22px;
+    font-size: 10pt;
+}
+
+/* Functionality groups in coverage matrix */
+.entity-group {
+    page-break-before: always;
```

Stylelint (17.14.0)

[error] Expected "page-break-before" to be "break-before" (property-no-deprecated)

(property-no-deprecated)